### PR TITLE
Fix/tao 3944 back button

### PR DESCRIPTION
**Pull request by jsconan**
Related to: https://oat-sa.atlassian.net/browse/TAO-3944

Requires: https://github.com/oat-sa/extension-tao-proctoring/pull/302

Add a controller to access the proctoring history, and refactor a little the existing controllers, in order to share common features.
Override the service `TestSessionHistoryService` to provide the right URL for the history page and the related back URL.

As the routes within the LTI proctoring are built on top of those from the regular proctoring extension, I had to override the controllers of the proctoring. This is needed because the routes handled by the LTI can be reached directly, and if they don't exists the app will be broken.
I did not find a better way to make this working, since the LTI just wraps the regular proctoring extension.

### CODE REVIEW

**Comment by jsconan** [comment]
@antoinerobin please validate the PHP code
@krampstudio please review the frontend side

File: views/js/controller/routes.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+define({
+    'Monitor' : {
+        'actions' : {
+            'index' : 'controller/monitoring'
```
Comment (krampstudio) [inline views/js/controller/routes.js]: Instead you can just point to the desired controller : 
```js
 'index' : 'taoProctoring/controller/Delivery/monitoring'
```
etc.

Comment (jsconan) [inline views/js/controller/routes.js]: Yes, that is what I did in first place. But the compilation just provided a very small minified file that only contains address of the target controllers. Is this ok?

Comment (krampstudio) [inline views/js/controller/routes.js]: Yes, they will be loaded from the target extension 'controllers.min.js'

Comment (krampstudio) [inline views/js/controller/routes.js]: But try just to be sure this still works with the new routing

Comment (jsconan) [inline views/js/controller/routes.js]: Ok, fixed!

**Comment by krampstudio** [comment]
Waiting for @antoinerobin approval to merge

**Pull request merged**